pkg/volume/quobyte/quobyte_test.go

```diff
@@ -91,13 +91,15 @@ func doTestPlugin(t *testing.T, spec *volume.Spec) {
 
 	pod := &v1.Pod{ObjectMeta: metav1.ObjectMeta{UID: types.UID("poduid")}}
 	mounter, err := plug.(*quobytePlugin).newMounterInternal(spec, pod, mount.NewFakeMounter(nil))
+
+	if mounter == nil {
+		t.Fatal("Got a nil Mounter")
+	}
+
 	volumePath := mounter.GetPath()
```

Comment:
this works but I would prefer to move this line after the error checking, so that we check all the errors first.

Reply:
oh, I didn't notice that at the time and thanks for your advice.

Reply:
I don't think I need to remove this line of code, since all the other tests judged `mounter == nil`.
For example:
https://github.com/kubernetes/kubernetes/blob/b0abe89ae259d5e891887414cb0e5f81c969c697/pkg/volume/cephfs/cephfs_test.go#L80-L85

If removed, do they need to be consistent?

Reply:
I think checking for nil is still good. But also looks like the other tests need to be fixed too?

Reply:
I misunderstood you, you said 'move', not 'remove'.  I misspelled 'remove'

Other test code locations should not need to be moved.

For the rest of the test code, I think it should look like this:
if mounter == nil {

// Use t.Fatal () instead of t.Errorf ()

}